frontend/sporting-milestones-timeline/style.css

```
@@ -0,0 +1,392 @@
+    --sm-bg-dark: #f8fafc;
+    --sm-card-bg: rgba(255, 255, 255, 0.92);
+    --sm-card-border: rgba(0, 0, 0, 0.1);
+    --sm-text-primary: #0f172a;
+    --sm-text-secondary: #475569;
+}
+
+.milestones-timeline-main {
+    background-color: var(--sm-bg-dark);
+    color: var(--sm-text-primary);
+    min-height: 100vh;
+    font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)